modules/eco-configuring-machine-health-check-with-poison-pill.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="configuring-machine-health-check-with-poison-pill_{context}"]
+= Configuring machine health checks to use the Poison Pill Operator 
+
+Use the following procedure to configure the machine health checks to use the Poison Pill Operator as a remediation provider.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `PoisonPillRemediationTemplate` CR
```

Comment:
This should have a colon at the end of the sentence to match elsewhere in the assembly.